compatibility/compatibility-vec12to20/src/main/java/com/foursoft/harness/compatibility/vec12to20/Vec12XTo20XClassMapper.java

```diff
@@ -32,17 +32,14 @@
 import com.foursoft.harness.vec.v2x.VecCableLeadThrough;
 
 import java.io.Serial;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
 
 /**
  * Class responsible for mapping VEC 1.2.X classes to VEC 2.X.X <b>and vice versa</b>.
  */
 public class Vec12XTo20XClassMapper extends NameBasedClassMapper {
 
-    private final Map<Class<?>, Class<?>> classMap;
+    private final Map<Class<?>, Class<?>> explicitClassMappings;
```

Comment:
@DManstrator give at consice name that describes what it is for instead of generic one.